readme.md

```diff
@@ -436,7 +436,7 @@
 - [CDK](https://github.com/kolomied/awesome-cdk#readme) - Open-source software development framework for defining cloud infrastructure in code.
 - [IAM](https://github.com/kdeldycke/awesome-iam#readme) - User accounts, authentication and authorization.
 - [Slim](https://github.com/nekofar/awesome-slim#readme) - PHP framework.
-
+- [Fiber](https://github.com/gofiber/awesome-fiber#readme) - Go framework.
```

Comment:
Needs a newline after the list entry. And a more detailed description, maybe something like: 
`- [Fiber](https://github.com/gofiber/awesome-fiber#readme) - Web framework built on top of Fasthttp, the fastest HTTP engine for Go.
`